### PR TITLE
fix for CVE-2020-26160

### PR DESCRIPTION
**Pull request by defbin**
[CVE-2020](https://jira.percona.com/browse/CVE-2020) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

https://github.com/dgrijalva/jwt-go/issues/482#issuecomment-950205708

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -35,6 +36,7 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible // Required by OLM
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt/v4 v4.1.0
```
Comment (delgod) [inline go.mod:39]: why not?
```suggestion
	github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt/v4 v4.2.0
```

**Review by delgod — CHANGES_REQUESTED** [review]

**Review by delgod — APPROVED** [review]

**Comment by defbin** [comment]
@delgod @hors upgraded minimal go version to 1.16 to avoid issue described in https://github.com/dgrijalva/jwt-go/issues/216

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/827/commits/8c8e4da3cd8ad272ea252c723ad3de7b2abd1680
image: `perconalab/percona-server-mongodb-operator:PR-827-8c8e4da3`

**Review by delgod — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/827/commits/8c8e4da3cd8ad272ea252c723ad3de7b2abd1680
image: `perconalab/percona-server-mongodb-operator:PR-827-8c8e4da3`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| scheduled-backup | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| demand-backup-sharded | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| users | passed |
| pitr | passed |
| data-sharded | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/827/commits/ceb446026305b9b62443ff9ab7d404f455ec1305
image: `perconalab/percona-server-mongodb-operator:PR-827-ceb44602`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| scheduled-backup | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| liveness | passed |
| demand-backup-sharded | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| data-sharded | passed |
| non-voting | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/827/commits/0544afcb9b1393f0fb364b5ad0c1f19b6367a41b
image: `perconalab/percona-server-mongodb-operator:PR-827-0544afcb`

**Review by delgod — APPROVED** [review]

**Pull request merged**